extensions/postgis/Makefile.in

```diff
@@ -65,8 +65,7 @@ CURV_big=@POSTGIS_MAJOR_VERSION@@POSTGIS_MINOR_VERSION@
 
 all: sql/$(EXTENSION)--$(EXTVERSION).sql sql/$(EXTENSION)--unpackaged--$(EXTVERSION).sql
 
-sql/$(EXTENSION).sql: $(EXTENSION_SCRIPTS)
-	mkdir -p sql
+sql/$(EXTENSION).sql: $(EXTENSION_SCRIPTS) | sql
```

Comment:
I've never seen this pipe notation before. Out of curiousity do you have a manual link I could read?

Reply:
https://www.gnu.org/software/make/manual/html_node/Prerequisite-Types.html

we're heavily using them in our processing pipelines. we also have a tool for that :)
https://github.com/gojuno/make-profiler